ddp-workspace/projects/ddp-dsm-ui/src/app/test-dss/test-dss.component.html

```diff
@@ -0,0 +1,14 @@
+<h2>test-dss works!</h2>
+<hr>
+
+<button (click)="createActivity()">CREATE</button>
+
+<button (click)="getActivity()">GET</button>
+
+<button (click)="fetchActivity()">FETCH</button>
+
+<!--<ddp-activity-redesigned [studyGuid]="studyGuid"-->
+<!--                         [activityGuid]="activityGuid"-->
+<!--                         (submit)="navigate($event)"-->
+<!--                         (activityCode)="activityCodeChanged($event)">-->
+<!--</ddp-activity-redesigned>-->
```

Comment:
Should work now for existing `userGuid`, `studyGuid`, `activityGuid` (they are set up in test-dss.component.ts)